### PR TITLE
add my more recent changes back and add script option

### PR DESCRIPTION
**Pull request by EdwardAngert**
Closes: #

## Summary

**[Doc Title](https://pantheon.io/docs/doc-title)** - A one sentence summation of the pertinent changes (including not-yet-completed work) provided by this PR.

## Effect

The following changes are already committed:

-
-

## Remaining Work

The following changes still need to be completed:

- [ ] List any outstanding work here

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/docs/old-path/` => `/docs/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] For Heroes - add a props post to the [discussion board](https://discuss.pantheon.io/c/pantheon-platform/documentation/17).
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | [Live](https://pantheon.io/docs) | [pr-6302](https://pr-6302--pantheon-docs.my.pantheonfrontend.website/docs) | Difference |
| --- | --- | --- | --- |
| Performance Score | `61` | `63` | `2` |
| Best Practice Score | `79` | `79` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-6302` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/26381/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/26381/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)


[Preview Environment](https://pr-6302--pantheon-docs.my.pantheonfrontend.website/docs)

File: source/content/guides/composer-convert.md [hunk 1]
```diff
@@ -12,151 +12,142 @@ reviewed: "2021-03-10"
...
+
+   - If there are settings from `pantheon.yml` (shown with a `-` in the diff output), consider copying over your old `pantheon.yml` to preserve these settings:
+
+     ```bash{promptUser:user}
+     git checkout master pantheon.yml
+     git add pantheon.yml
+     git commit -m 'Copy my pantheon.yml'
+     ```
+
+   If you prefer to keep the value for `database` from `pantheon.upstream.yml`, remove it from `pantheon.yml`.
```
Comment (greg-1-anderson) [inline source/content/guides/composer-convert.md:150]: Most users will not have a `database` entry in their pantheon.yml file. The advice we give here varies based on when this guide is published. If the restriction on Mariadb upgrades has been lifted, then we should not have users modifying the database entry at all. If someone wants to convert before the restriction has been lifted, then they *must* override the database entry in their pantheon.yml and pin it back to 10.0, but if they do this then they *must* come back in the future and remove their pin.

Given the timeframe, it's probably best to wait until the restriction is lifted before publishing.

File: source/content/guides/composer-convert.md [hunk 2]
```diff
@@ -166,119 +157,130 @@ What makes your site code unique is your selection of contributed modules and th
...
 
-A Composer-managed site should be able to include all custom code via Composer. Begin by reviewing your existing site's code. Check for contributed modules in `/modules`, `/modules/contrib`, `/sites/all/modules`, and `/sites/all/modules/contrib`.
+Begin by reviewing the existing site's code. Check for contributed modules in `/modules`, `/modules/contrib`, `/sites/all/modules`, and `/sites/all/modules/contrib`.
 
-1. When reviewing the site, take stock of exactly what versions of modules and themes you depend on. One way to do this is to use a command like the following from within a contributed modules folder (e.g. `/modules`, `/themes`, `/themes/contrib`, `/sites/all/themes`, `/sites/all/themes/contrib`, etc.).
+1. When reviewing the site, take stock of exactly what versions of modules and themes you depend on. One way to do this is to change to run the `pm:projectinfo` Drush command from within a contributed modules folder (e.g. `/modules`, `/themes`, `/themes/contrib`, `/sites/all/themes`, `/sites/all/themes/contrib`, etc.).
```
Comment (greg-1-anderson) [inline source/content/guides/composer-convert.md:168]: Drush `pm:projectinfo` does not care what your cwd is. This command reads module information from the database, so it won't work on the local site unless the user has also set up a local database. This is why the recommended command below uses `terminus drush` & runs against the dev environment.

File: source/content/guides/composer-convert.md [hunk 3]
```diff
@@ -166,119 +157,130 @@ What makes your site code unique is your selection of contributed modules and th
...
 
-To run the module migration script, cd to the root directory of your repository and run `bin/migrateModules.php`.
+If you still want to try it:
 
-You can see which modules were added by running `git diff composer.json`.
+1. Say the following out loud: "Pantheon is not responsible for what I am about to do."
 
-Then remove the `composer.lock` file (you can delete it. Composer will create a new one with the modules you just added) and run `composer install` and resolve any lingering version conflicts.
-</Alert>
+1. Create a directory called `bin` within your repository and copy the `migrateModules.php` file from [GitHub](https://github.com/pantheon-upstreams/drupal-project/pull/8/files#diff-c68470275758ca395b98d53ed258b63519435d492dd51531c4cf372814c6593e) into that directory.
```
Comment (greg-1-anderson) [inline source/content/guides/composer-convert.md:238]: We should publish this script somewhere else. We could tell them to install via `composer require` if we published on Packagist.

I still think we should test and publish the docs without any scripts first, and then add these instructions as a follow-on improvement.

**Review by stovak — APPROVED** [review]

**Pull request merged**